### ISSUE TITLE
training dataset problem

### ISSUE THREAD

**Issue by hannah4kr** [clovaai/deep-text-recognition-benchmark#346]
Hi, 

I would like to make the model working well on both printed and handwritten papers. I am confused about the way to train the model.
So, if I want the model to recognize both printed characters and handwritings, do I need to train both printed and handwriting characters at the same time? Or should I train them separately? 
This is my first time training the OCR model, so please help me in details if possible.

Thank you in advance!

**Comment by gitdeepheolp** [comment]
i think you can train them separately. use the --FT option second time.

**Comment by hannah4kr** [comment]
I got it!
Another question is that, if I train the model with only characters, how does it recognize words?

**Comment by gitdeepheolp** [comment]
Are you sure that you trained the model only with characters?

**Comment by hannah4kr** [comment]
Yes, the model works well on recognizing each character, but not words.
Is it because I did not use the proper dataset for training?

**Comment by hannah4kr** [comment]
@ClovaAIAdmin any suggestion?
